storybook/config/preview.js

```diff
@@ -64,16 +64,18 @@ export const parameters = {
     },
     docs: {
         // Customize docs page layout (in order to rename 'Stories' section)
-        page: () => (
-            <>
-                <Title />
-                <Subtitle />
-                <Description />
-                <Primary />
-                <ArgsTable story={PRIMARY_STORY} />
-                <Stories title="Examples" />
-            </>
-        ),
+        page: function Page() {
```

Comment:
Out of curiousity, why the change to a named function ?

Reply:
I got a prettier error.. Dunno why

Reply:
Huh.